panel/src/components/Sections/ModelsSection.vue

```diff
@@ -120,6 +172,9 @@ export default {
 		canSearch() {
 			return this.options.search;
 		},
+		canSelect() {
+			return this.options.batch && this.items.length > 0;
```

Comment:
`Question:` Should we offer selection mode for sections with a single item?

Reply:
Probably not. It feels pretty weird.

Reply:
Does it do harm? I think for consistency it would still be useful. E.g. a user wants to delete all items and is used to using the batch editing mode for this. They can still use it with just one item. If we just hide the button, the user could be confused.

Reply:
I've thought about it again and with the opt-in character of the feature I start to think that we should just keep it, even if there's only one option.